src/app/components/Header/Header.tsx

```diff
@@ -1,14 +1,89 @@
 "use client";
+import { useState } from "react";
 import { styles } from "./Header.style";
-import { Box, Typography } from "@mui/material";
-import { HeaderProps } from "@/app/general/interfaces";
+import { Box, Typography, Button } from "@mui/material";
+import Table from "@/app/components/Table";
+import Dialog from "@/app/components/Dialog";
+import {
+    HeaderProps,
+    NameAndDescription,
+    generalObject,
+} from "@/app/general/interfaces";
+import { strOrNum } from "@/app/general/types";
+
+export default function Header({ bot }: HeaderProps) {
+    const [openAttribute, setOpenAttribute] = useState<boolean>(false);
+    const [openData, setOpenData] = useState<boolean>(false);
+
+    const botHeaders = bot?._data.headers;
+    const botColumns = bot?._data.columns;
+
+    const rows: generalObject<strOrNum>[] = [];
+
+    [0, 1, 2, 3, 4].forEach((index) => {
+        const row: generalObject<strOrNum> = {};
+        botColumns.forEach((col) => {
+            row[col.displayName] = col.rows[index];
+        });
+        rows.push(row);
+    });
+
+    const attributesRows = botColumns?.map((column) => ({
+        name: column.displayName,
+        description: column._description,
+    }));
 
-export default function Header({ text }: HeaderProps) {
     return (
         <Box component="header" sx={styles.box}>
-            <Typography variant="h4" component="h1">
-                {text}
-            </Typography>
+            <Box component="div" sx={styles.buttonContainer}>
+                <Button
```

Comment:
Consider saving components like that in another element; if you have several buttons you use and set all the parameters every time, it is a way for the typo that can harm consistency in the UI. Still, If you create and use a custom button with all the attributes, you fix it. Don't do that now, but it opens an issue with low priority for generalise ui components like the button.